tdp/cli/commands/deploy.py

```diff
@@ -32,11 +29,22 @@
     is_flag=True,
     help="Force stale status update.",
 )
-@dry
+@click.option("--dry", is_flag=True, help="Execute dag without running any action")
 @collections
 @database_dsn
-@mock_deploy
-@run_directory
+@click.option(
+    "--mock-deploy",
+    envvar="TDP_MOCK_DEPLOY",
+    is_flag=True,
+    help="Mock the deploy, do not actually run the ansible playbook",
+)
+@click.option(
+    "--run-directory",
+    envvar="TDP_RUN_DIRECTORY",
+    type=Path,
```

Comment:
the rest looks ok to me
tested ok

Reply:
> Use `click.Path(resolve_path=True, path_type=Path, exists=True)` instead of `Path` to correctly raise an exception when wrong arguments.

Done. I opened a follow up issue here: #488